docs/use_cases/model_applications/s2s_soil_moisture/GridStat_fcstSFSGSL_obsERA5Land_SoilMoisture.py

```diff
@@ -0,0 +1,140 @@
+#
+# .. highlight:: bash
+# .. literalinclude:: ../../../../parm/met_config/GridStatConfig_wrapped
+#
+
+##############################################################################
+# Running METplus
+# ---------------
+#
+# This use case can be run two ways:
+#
+# 1) Passing in GridStat_fcstSFSGSL_obsERA5Land_SoilMoisture.conf then a user-specific system configuration file::
+#
+#        run_metplus.py /path/to/METplus/parm/use_cases/model_applications/s2s_soil_moisture/GridStat_fcstSFSGSL_obsERA5Land_SoilMoisture /path/to/user_system.conf
+#
+# 2) Modifying the configurations in parm/metplus_config, then passing in GridStat_fcstSFSGSL_obsERA5Land_SoilMoisture::
+#
+#        run_metplus.py /path/to/METplus/parm/use_cases/model_applications/s2s_soil_moisture/GridStat_fcstSFSGSL_obsERA5Land_SoilMoisture.conf
+#
+# The former method is recommended. Whether you add them to a user-specific configuration file or modify the metplus_config files, the following variables must be set correctly:
+#
+# * **INPUT_BASE** - Path to directory where sample data tarballs are unpacked (See Datasets section to obtain tarballs). This is not required to run METplus, but it is required to run the examples in parm/use_cases
+# * **OUTPUT_BASE** - Path where METplus output will be written. This must be in a location where you have write permissions
+# * **MET_INSTALL_DIR** - Path to location where MET is installed locally
+#
+# Example User Configuration File::
+#
+#   [config]
+#   INPUT_BASE = /path/to/sample/input/data
+#   OUTPUT_BASE = /path/to/output/dir
+#   MET_INSTALL_DIR = /path/to/met-X.Y 
+#
+#
+
+##############################################################################
+# Expected Output
+# ---------------
+#
+# A successful run will output the following both to the screen and to the logfile::
+#
+#   INFO: METplus has successfully finished running.
+#
+# Refer to the value set for **OUTPUT_BASE** to find where the output data was generated.
+# Output for the use case will be found in 30 folders(relative to **OUTPUT_BASE**).
+# The output will follow the time information of the run. Specifically:
+#
+# * YYYY060100
+#
+# where YYYY will be replaced by values corresponding to each of the years (1991 through 2020).
+# Each of those folders will have the following files:
+#
+# * grid_stat_SFS-GSL_vs_ERA5_060000L_YYYY0601_000000V_fho.txt
+# * grid_stat_SFS-GSL_vs_ERA5_060000L_YYYY0601_000000V_pairs.nc 
+# * grid_stat_SFS-GSL_vs_ERA5_060000L_YYYY0601_000000V.stat
+#
+
+##############################################################################
+# Keywords
+# --------
+#
+# .. note::
+#
+#   * GridStatToolUseCase
+#   * PythonEmbeddingFileUseCase
+#   * S2SAppUseCase
+#   * NETCDFFileUseCase
+#
+#   Navigate to the :ref:`quick-search` page to discover other similar use cases.
+#
+#
+#
+# sphinx_gallery_thumbnail_path = '_static/s2s-GridStat_fcstSFSGSL_obsERA5Land_SoilMoisture.png'
```

Comment:
This is an incorrect file name; it should include `s2s_soil_moisture` as the use case group and then it should render the right image